kotlin-inject-runtime/src/commonMain/kotlin/me/tatarka/inject/annotations/Annotations.kt

```diff
@@ -26,4 +26,7 @@ annotation class IntoSet
 annotation class IntoMap
 
 @Target(VALUE_PARAMETER)
-annotation class Assisted
\ No newline at end of file
+annotation class Assisted
+
+@Target(FUNCTION)
+annotation class TargetComponentAccessor
```

Comment:
Finding a good name for this is tricky...

If my understanding is correct, the method is a way to reference the actual create function for a component from the shared source set.  How about `ReferenceActualComponent`?

Reply:
I was approaching it more as a way to access the create function that is generated in each target's source set from the shared source set. So it doesn't really reference it as much as call through to it.